definitions/divorce/json/CaseTypeTab/CaseTypeTab-prod.json

```diff
@@ -99,5 +99,15 @@
     "TabDisplayOrder": 20,
     "CaseFieldID": "CaseNotes",
     "TabFieldDisplayOrder": 1
+  },
```

Comment:
feature toggle please :)

Reply:
Yes, I know we have "global" `prod` already. But we want to move away from this approach. It's very confusing when release.